### PR TITLE
185 sort catalog properties

### PR DESCRIPTION
**Pull request by Andrew-Orlov**
https://app.zenhub.com/workspaces/vc-operations-564a7a2e8294d8de6364f3d7/issues/virtocommerce/vc-module-catalog/185

### CODE REVIEW

File: VirtoCommerce.CatalogModule.Data/Services/PropertyDictionaryItemService.cs [hunk 1]
```diff
@@ -121,7 +121,7 @@ public GenericSearchResult<PropertyDictionaryItem> Search(PropertyDictionaryItem
                     var sortInfos = criteria.SortInfos;
                     if (sortInfos.IsNullOrEmpty())
                     {
-                        sortInfos = new[] { new SortInfo { SortColumn = "SortOrder" } };
+                        sortInfos = new[] { new SortInfo { SortColumn = "SortOrder", SortDirection = SortDirection.Descending } };
```
Comment (artem-dudarev) [inline VirtoCommerce.CatalogModule.Data/Services/PropertyDictionaryItemService.cs]: Undo. This is incorrect.

Comment (artem-dudarev) [reply VirtoCommerce.CatalogModule.Data/Services/PropertyDictionaryItemService.cs]: Not descending!

Comment (Andrew-Orlov) [reply VirtoCommerce.CatalogModule.Data/Services/PropertyDictionaryItemService.cs]: Changed to ascending

**Comment by Andrew-Orlov** [comment]
Should it be described in the task. Maybe.

**Comment by artem-dudarev** [comment]
The issue description doesn't require to sort with descending order. So there was no need to introduce such sorting.

**Pull request merged**